### DISCUSSION TITLE
Error: No such built-in module: node:stream/promises

### DISCUSSION THREAD

**Discussion by AxelBriche** [nuxt/nuxt#16854] [Questions]
<img width="776" alt="CleanShot 2022-09-16 at 21 32 11@2x" src="https://user-images.githubusercontent.com/1295251/190716595-d1e869bc-97b5-466f-b28e-784b1b0a8932.png">

Hello, how to correct this error please? `No such built-in module: node:stream/promises`

**Comment by pi0** [comment]
It will be fixed in next version. Please use Node.js 16 (latest LTS) to resolve.
